VARIABLES.md

```diff
@@ -4,6 +4,14 @@ Below are the supported variables for the role confluent.variables
 
 ***
 
+### mask_secrets
+
+Boolean to mask secrets in playbook output
+
+Default:  true
+
+***
+
```

Comment:
rerun `python docs.py` i expect to see mask_diff here

Reply:
Done!